Keycloak/clients.dat

```diff
@@ -0,0 +1,8794 @@
+[
+  {
+    "id": "7b7b4724-2342-4bfa-8779-17ee08144873",
+    "clientId": "account",
```

Comment:
We are only interested in GRAD clients. Not ALL the clients. Please remove any client that is not part of GRAD. You have included system clients etc which we don't need for this.